apps/portal/src/content/docs/components/alert-dialog.mdx

```diff
@@ -0,0 +1,341 @@
+---
+title: AlertDialog
+description: AlertDialog component documentation
+---
+
+The `AlertDialog` component provides a flexible UI element for displaying alert dialogs. It is composed of several subcomponents such as `AlertDialogHeader`, `AlertDialogFooter`, `AlertDialogTitle`, and `AlertDialogDescription` to offer a structured and customizable interface.
+
+import { DocsPage } from "@/components/docs-page";
+import { AlertDialog } from "@harnessio/ui/components";
+import { Button } from "@harnessio/ui/components";
```

Comment:
You don't need to import these unless they're being used in the MDX itself. The `ComponentExample` imports everything it needs to render and `ui` lib component.
```suggestion
```